CONTRIBUTING.md

```diff
@@ -94,14 +94,36 @@ open http://localhost:8000/
 
 To publish the packages to npm, you'll need to be added as an owner for the
 packages you're publishing. Use the #design-system Slack channel for more
-information.
+information. 
+
+Before Publishing:
+
+- Know that publishing is very easy once you have access to do so **Please Be Careful** 🤗
+- As of v2 you must be on Node v8+
+- name sure you are logged into NPM properly, if you have multiple logins you can use something like [npmrc]
 
 These Lerna commands can be helpful when publishing:
 
 - Run `npm run changed` to see which packages have changed since the last
   release.
 - Run `npm run diff` to see a diff of all packages since the last release.
 - Run `npm run publish` to publish the updated packages with Lerna.
+- Do not run `npm publish` on it's own.
+
+To set the packages to the same SemVer/custom tag run `npm run publish` and follow the prompts from the lerna-cli.  
```

Comment:
Tiny nit: if you like to break line when > 80 characters, think you can start with a space in 2nd line in markdown, and the formatted page would still display the line without breaking.